chat-client/src/client/messager.ts

```diff
@@ -179,16 +178,15 @@ export class Messager {
         this.chatApi.telemetry({ ...params, name: ERROR_MESSAGE_TELEMETRY_EVENT })
     }
 
-    onOpenTab = (result: OpenTabResult | ErrorResult): void => {
-        this.chatApi.onOpenTab(result)
+    onOpenTab = (requestId: string, result: OpenTabResult | ErrorResult): void => {
```

Comment:
Can we also add a test that `requestId` was propagated from inbound to outbound message? Can be separate PR.